docs/docs/spec/packaging/guidelines.md

```diff
@@ -19,6 +27,14 @@ Examples of both profile options usage for the full docker image name and only t
 
 **AVOID** using the default tag `latest`. This might bring issues among different package versions. You **SHOULD** use unique tags that identify a fixed version of the containerized application you want the user to run.
 
+### Why docker image builds are not supported
+
+The issues around building docker images can be summarized:
+
+- **Introduce vulnerabilities and security concerns** to the user’s host machine because **builds could execute arbitrary and potentially malicious code**. Currently, the attack surface is quite extensive as anyone with write access to the AVS Tap repo could add this type of code and change the AVS Node docker build pipeline or Docker Compose, create a git tag and make a release.
+- Supporting docker image builds **involves cloning and using the entire repository** where the AVS package lies. This means not only that we are potentially copying a lot of unnecessary data for the user into his/her disk and making the installation process heavier than necessary, but also that we can’t control the usage of good practices around the AVS packaging. If, for example, we would like to enforce creating a separate repository for the AVS package only, it wouldn’t be possible if we allow docker builds.
+- Depending on docker builds for the users to use your software **is not a good practice**. An AVS developer will likely push the Docker images to a Docker registry. Maintaining two different ways of using your official Docker Compose script is a burden, as there are differences between using already-built images and building images from a local/remote context. Building the docker image could be helpful for development environments. However, it **is not expensive to build your image** before running your Compose script and target the local image inside it. AVS developers could add a simple `docker build` command to their build scripts or E2E test setup parts, use an env variable for the `image` field of the Compose script, and only change the env variable value. This introduces a new problem: which tag to use for the docker image. We are going to cover that in the next section.
```

Comment:
should we link the mock AVS here as an example of how to do this?

Reply:
Done [135b212](https://github.com/NethermindEth/el-node-spec/pull/54/commits/135b2127e475cca245ae8279bc90646408862b7d)

Reply:
Done [bf17df6](https://github.com/NethermindEth/el-node-spec/pull/54/commits/bf17df6ab4ec80310f5abba0bde3b7b9b00a72f4)